resources/class_error.php

```diff
@@ -31,7 +31,7 @@ function __construct()
 	 * @param integer $line The error line
 	 * @return boolean True if parsing was a success, otherwise assume a error
 	 */
-	function error($type, $message, $file=null, $line=0)
+	function error($type, $message, $file=null, $line=0, $allow_output=false)
```

Comment:
Here the default value for `$allow_output` might be better to set to `true`, and the PHPDoc needs to be updated.